### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by timostamm**
This updates the top-level README to better reflect that this repository now hosts an implementation for web clients as well as for Node.js.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,65 @@
...
-// {sentence: 'When you feel happy, what do you do?'}
+```
+service ElizaService {
+  rpc Say(SayRequest) returns (SayResponse) {}
+}
 ```
 
-Using the client generated by Connect-Web, the same call becomes quite a bit 
-simpler:
+With the magic of code generation, this schema produces servers and clients:
```
Comment (smaye81) [inline README.md:18]: I think `And with the magic of code generation...` sounds a bit better maybe.

File: README.md [hunk 2]
```diff
@@ -1,66 +1,65 @@
...
+    --data '{"sentence": "I feel happy."}' \
+    https://demo.connect.build/buf.connect.demo.eliza.v1.ElizaService/Say
+```
 
+With Connect for ECMAScript, you can spin up a service in Node.js and call it
+from the web, the terminal, or native mobile clients. Under the hood, it uses 
+[Protocol Buffers](https://github.com/bufbuild/protobuf-es) for the schema, and 
+implements RPC (remote procedure calls) with three protocols: The widely available 
+gRPC and gRPC-web, and Connect's [own protocol](https://connect.build/docs/protocol/), 
+optimized for the web. This gives you unparalleled interoperability with type-safety end
```
Comment (smaye81) [inline README.md:42]: Since you used `end to end` above, maybe something like `...interoperability with full-stack type-safety`?

File: README.md [hunk 3]
```diff
@@ -1,66 +1,65 @@
...
+With Connect for ECMAScript, you can spin up a service in Node.js and call it
+from the web, the terminal, or native mobile clients. Under the hood, it uses 
+[Protocol Buffers](https://github.com/bufbuild/protobuf-es) for the schema, and 
+implements RPC (remote procedure calls) with three protocols: The widely available 
+gRPC and gRPC-web, and Connect's [own protocol](https://connect.build/docs/protocol/), 
+optimized for the web. This gives you unparalleled interoperability with type-safety end 
+to end.
 
-## Connect for Node.js (Preview)
+To get started, head over to the [docs](https://connect.build/docs/web/getting-started)
```
Comment (smaye81) [inline README.md:45]: Should we link directly to the `web` docs?  Maybe we should have two links here for Node and Web?

File: README.md (hunk 1 above)
Comment (timostamm) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (timostamm) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -73,19 +72,20 @@ happy to chat!
...
-* [connect-crosstest](https://github.com/bufbuild/connect-crosstest):
-  gRPC-Web and Connect interoperability tests
+We support all modern web browsers that implement the widely available 
+[fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)
+and the [Encoding API](https://developer.mozilla.org/en-US/docs/Web/API/Encoding_API).
+The libraries and the generated code are compatible with ES2017 and TypeScript 4.1.
+
+On Node.js, we use the `http`, `https`, or `http2` modules, and support v16, v17 and v18.
+
+Would like to use Connect on other platforms like Bun, Deno, Vercel’s Edge Runtime,
```
[the remark this answers is not in the document]
Comment (timostamm) [reply README.md]: TY, fixed!

**Pull request merged**